Makefile

```diff
@@ -46,22 +46,17 @@ ARGS ?= $(EXTRA_ARGS)
 .DEFAULT_GOAL := help
 
 .PHONY: all
-all: clean dep verify build install ## Test, build, install
+all: clean verify build install ## Test, build, install
 	@echo "+ $@"
 
 .PHONY: init
-init: ## Initializes this Makefile dependencies: dep, golint, staticcheck, checkmake
+init: ## Initializes go tools this Makefile uses: golint, staticcheck, goimports, checkmake
```

Comment:
```
init: ## Initializes go tools: golint, staticcheck, goimports, checkmake
```

Reply:
less info = more sense? ;)

Reply:
okay, so I think you forgot a comma between `go tools` and `this Makefile uses`